README.md

```diff
@@ -0,0 +1,61 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
+
+> This project is done as part of the 2nd day activities. It is made in order to help the student masters the tools and practice needed for all of followings projects in Microverse curriculum
+
+![screenshot](app_screenshot.png)
+
+Additional description about the project and its features.
+
+## Built With
+
+-HTML
+-CSS
+-Microverse html/css linter and markdown template
+
+## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+Verify you install git (if not istall it)
+
+### Setup
+
+Set up git if needed
+
+### Clone
```

Comment:
- [ ] [Optional] Please edit out these sections to reflect what your project is about. You can also opt to leave them out.